src/SFML/Graphics/CMakeLists.txt

```
@@ -97,26 +98,57 @@ target_include_directories(sfml-graphics SYSTEM PRIVATE "${PROJECT_SOURCE_DIR}/e
 # glad sources
 target_include_directories(sfml-graphics SYSTEM PRIVATE "${PROJECT_SOURCE_DIR}/extlibs/headers/glad/include")
 
-# let CMake know about our additional graphics libraries paths
-if(SFML_OS_WINDOWS)
-    list(APPEND CMAKE_INCLUDE_PATH "${PROJECT_SOURCE_DIR}/extlibs/headers/freetype2")
-elseif(SFML_OS_MACOS)
-    list(APPEND CMAKE_INCLUDE_PATH "${PROJECT_SOURCE_DIR}/extlibs/headers/freetype2")
-    list(APPEND CMAKE_LIBRARY_PATH "${PROJECT_SOURCE_DIR}/extlibs/libs-macos/Frameworks")
-elseif(SFML_OS_IOS)
-    list(APPEND CMAKE_INCLUDE_PATH "${PROJECT_SOURCE_DIR}/extlibs/headers/freetype2")
-elseif(SFML_OS_ANDROID)
-    list(APPEND CMAKE_INCLUDE_PATH "${PROJECT_SOURCE_DIR}/extlibs/headers/freetype2")
-endif()
-
 # find external libraries
 if(SFML_OS_ANDROID)
     target_link_libraries(sfml-graphics PRIVATE z)
 elseif(SFML_OS_IOS)
     target_link_libraries(sfml-graphics PRIVATE z bz2)
 endif()
 
-find_package(Freetype REQUIRED)
+if(SFML_USE_SYSTEM_DEPS)
+    find_package(Freetype REQUIRED)
+else()
+    # use an immediately invoked function to scope option variables we have to set
+    function(sfml_add_graphics_dependencies)
+        include(FetchContent)
+
+        # remember whether we are building SFML as a shared library
+        if(BUILD_SHARED_LIBS)
+            set(SFML_BUILD_SHARED_LIBS ON)
+        endif()
+
+        set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+        set(BUILD_SHARED_LIBS OFF)
+        set(FT_DISABLE_ZLIB ON)
+        set(FT_DISABLE_BZIP2 ON)
+        set(FT_DISABLE_PNG ON)
+        set(FT_DISABLE_HARFBUZZ ON)
+        set(FT_DISABLE_BROTLI ON)
+        set(FT_ENABLE_ERROR_STRINGS ON)
+
+        FetchContent_Declare(Freetype
+            GIT_REPOSITORY https://github.com/freetype/freetype.git
+            GIT_TAG VER-2-13-2
+            GIT_SHALLOW ON)
+        FetchContent_MakeAvailable(Freetype)
+
+        set_target_properties(freetype PROPERTIES FOLDER "Dependencies")
```

Should this be capitalized?
```suggestion
        set_target_properties(Freetype PROPERTIES FOLDER "Dependencies")
```

---

The target name used by the FreeType CMake configuration is lowercase `freetype`.

---

Maybe, I'm misunderstanding something, but shouldn't it match the casing of the FetchContent call?